src/libstd/rt.rs

```diff
@@ -65,3 +66,10 @@ fn lang_start(main: *const u8, argc: isize, argv: *const *const u8) -> isize {
         0
     }
 }
+
+/// Fixed frame used to clean the backtrace with `RUST_BACKTRACE=1`.
+#[no_mangle]
+#[inline(never)]
+pub fn __rust_begin_backtrace_binary(f: fn()) {
```

Comment:
If possible, I would follow the example of `rust_panic` and use `#[no_mangle]` and `#[allow(private_no_mangle_fns)]`: https://github.com/rust-lang/rust/blob/a559452b05c20041a27f518d262573c56b876b64/src/libstd/panicking.rs#L574-L577